### ISSUE TITLE
[A11y] Cookies management panel `h1` always present in the page

### ISSUE THREAD

**Issue by hannahiss** [Orange-OpenSource/Orange-Boosted-Bootstrap#2862]
### Prerequisites

- [x] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [x] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [x] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

The `h1` title that is always present in the page (in `visually-hidden`), is apparently that of the banner, but this title is always in the page, even when the banner is not visible.

In addition, when we open the modal from this banner, we then have 2 identical `h1` titles "Cookies management panel": one for the banner and one for the modal.

So ideally, tarteaucitron should:

- put a display: none on the `h1` title when the banner is also hidden (and not just visually-hidden)
- differentiate the two `h1` titles, so that when they are both in the page, we can differentiate them, for example: "Cookies management banner" and "Cookies management panel (details)" or something like that...
- Be careful with the order in the code: to really do things well, the content of the banner should come after the title of the banner and the content of the modal should come after the title of the modal... Currently it seems a bit mixed when we have both displayed.

See: https://github.com/AmauriC/tarteaucitron.js/issues/1258 

Related to: https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2859

### Reduced test cases

Inspect the code to see the headings.

![Image](https://github.com/user-attachments/assets/12974f45-0f06-4e6f-b0e1-6b124e0d54fc)

Do it again when the banner is opened (if needed with `tarteaucitron.userInterface.openAlert()`).

![Image](https://github.com/user-attachments/assets/5c77157f-b529-4d85-834d-9ccc1c8dbe22)

Do it again with the banner and the modal opened (click on "Personalize")

![Image](https://github.com/user-attachments/assets/f6de4642-d8f8-4510-a735-ce937075491b)

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome

### What version of Boosted are you using?

5.3.3

**Comment by MaxLardenois** [comment]
This is partially fixed by https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/dd8cafa658c75bb67397f1aece7321d3f997e8b2
I will investigate and open an issue for the rest
